client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -121,6 +129,12 @@ $deisgn-selector-selection-space: 175px;
 	overflow: hidden;
 	padding: 2px;
 
+	&:hover,
+	&:focus {
+		border: 3px solid var( --color-neutral-dark );
```

Comment:
I'm a little surprised by Calypso color variable here instead of Gutenberg variable. Not a blocker for merging, just something we'll likely need to audit at some point.

Reply:
It's the color used in design and also in the current page template selector. The closest Gutenberg variable would be `$dark-gray-600: #40464d;` but it has less blue and more grey. 

I agree we need to define together with design team which parts should be using Gutenberg styles and where should come in Calypso theming. 

Also we should take into consideration that mixing color variables will increase the complexity of dark theme support ( https://github.com/Automattic/wp-calypso/issues/38587 )